packages/expect/src/__tests__/__snapshots__/spyMatchers.test.js.snap

```diff
@@ -127,44 +127,54 @@ Expected mock function to have been last called with:
 `;
 
 exports[`lastReturnedWith a call that throws is not considered to have returned 1`] = `
-"<dim>expect(</><red>jest.fn()</><dim>).lastReturnedWith(</><green>expected</><dim>)</>
+"<dim>expect(</><red>jest.fn()</><dim>).</>lastReturnedWith<dim>(</><green>expected</><dim>)</>
 
-Expected mock function to have last returned:
-  <green>undefined</>
-But the last call <red>threw an error</>"
+Expected value: <green>undefined</>
+Received value: threw an error
```

Comment:
Any chance we could make this "Received an error"?

Reply:
Ouch, the special case takes a step forward and backward.

In a case not in tests nor pictures, what do you think about the phrases when a call is implied:

```sh
Expected value: undefined
Received value
             1: has not returned yet
             2: threw an error
```

Reply:
Let’s see if we can improve the special strings independent of the labels.

It seems like the phrases need an explicit subject. I’m not sure if it is call or function.

Baseline was:
* `But the X call has not returned yet`
* `But the X call threw an error`

Reply:
Oh I see, it needs to be compatible after both `Received:` and `1:`.
What about
```
Received an error

Received value 42

Received no return yet

Received
      1: an error
      2: value 42
      3: no return yet

```

Although we'd probably need a special case for the very common case where every call returned:
```
Received value
            1: 42
            2: 1337
```

Reply:
Maybe I'm trying too hard to make everything a complete sentence when it's not even on the same line in the call list. Probably doesn't matter, the call list is super cool either way. @thymikee wdyt about the phrasing in this PR?

Reply:
@jeysal Do you think the additional call lines seem relevant when test fails? The most helpful question from the best reviewer of my technical writing was “why are you telling this?”

Whatever we do here, we can use as pattern in next PR for `toHaveBeen*Called` matchers.

Reply:
Yeah I think showing other calls around the call being asserted on and calls with equal values are good ideas